@zewelor Just a few last PyLint fixes needed:

```
./homeassistant/components/yeelight/light.py:85:1: D202 No blank lines allowed after function docstring
./homeassistant/components/yeelight/__init__.py:1:1: D200 One-line docstring should fit on one line with quotes
./homeassistant/components/yeelight/__init__.py:146:1: E303 too many blank lines (3)
./homeassistant/components/yeelight/__init__.py:147:1: D202 No blank lines allowed after function docstring
```

--- a/homeassistant/components/yeelight/__init__.py
+++ b/homeassistant/components/yeelight/__init__.py
@@ -1,4 +1,4 @@
-"""Support for Xiaomi Yeelight Wifi color bulb."""
+"""Support for Xiaomi Yeelight WiFi color bulb."""
 
 import logging
 from datetime import timedelta

--- a/homeassistant/components/yeelight/__init__.py
+++ b/homeassistant/components/yeelight/__init__.py
@@ -1,6 +1,5 @@
-"""
-Support for Xiaomi Yeelight Wifi color bulb.
-"""
+"""Support for Xiaomi Yeelight Wifi color bulb."""
+
 import logging
 from datetime import timedelta
 
@@ -142,10 +141,8 @@ def _parse_custom_effects(effects_config):
     return effects
 
 
-
 def setup(hass, config):
     """Set up the Yeelight bulbs."""
-
     conf = config[DOMAIN]
     yeelight_data = hass.data[DATA_YEELIGHT] = {}
 

--- a/homeassistant/components/yeelight/light.py
+++ b/homeassistant/components/yeelight/light.py
@@ -83,7 +83,6 @@ YEELIGHT_EFFECT_LIST = [
 
 def _cmd(func):
     """Define a wrapper to catch exceptions from the bulb."""
-
     def _wrap(self, *args, **kwargs):
         import yeelight
         try:

